lib/restpack_serializer/serializable.rb

```diff
@@ -1,4 +1,5 @@
 require 'active_support/concern'
+require "active_support/core_ext/string"
```

Comment:
Does this change belong in https://github.com/RestPack/restpack_serializer/pull/153 instead (for `String#camelize`)? I can't see how it's related to updating factory bot, and the tests pass locally for me without it.

Reply:
Whoops, yeah, it was for that one. That branch actually seems to pass fine without it anyway though so I've just removed it from here rather than also adding it there.